ada_feeding_msgs/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ rosidl_generate_interfaces(${PROJECT_NAME}
   "srv/GetSettings.srv"
   "srv/SetSettings.srv"
   "srv/ToggleFaceDetection.srv"
+  "srv/GetPoseStamped.srv"
```

Comment:
Are we using this anymore? If not, can you think of a future scenario where we may want to use it? If not, I'd suggest removing it.